common/src/main/java/org/opensearch/ml/common/PredictMode.java

```diff
@@ -0,0 +1,18 @@
+package org.opensearch.ml.common;
+
+import java.util.Locale;
+
+public enum PredictMode {
+    PREDICT,
+    BATCH,
+    ASYNC,
+    STREAMING;
```

Comment:
If we don't support this , let's add it later to avoid confusion

Reply:
commented out async and streaming for now.